### DISCUSSION TITLE
Resetting LOGB

### DISCUSSION THREAD

**Discussion by tetrahydra** [gopro/labs#440] [General]
Hi, I can't find any reference regarding resetting (removing) LOGB after it has been set to a certain value, for example, LOGB=400. Unlike WIDE=1, I can "reset" it to WIDE=0.

How do I reset/remove LOGB? Or in general, how do I "remove" certain settings without the need to !RESET?

Thank you.

**Comment by dnewman-gpsw** [comment]
!MLOGB=0 will reset it.  All !Mwxyz commands can be individual reset by setting them to zero.

**Reply by tetrahydra** [reply]
I have yet to test this, but once it is "reset", will the LOGB=0 be shown in the metadata? Because my concern is that, if it is not "removed" then it will still be executed during boot time.

**Reply by dnewman-gpsw** [reply]
Only non-zero values are applied.  LOGB=0 will appear in the metadata, but nothing is applied.
